### ISSUE TITLE
Authentication

### ISSUE THREAD

**Issue by geniosoft** [dougdellolio/coinbasepro-csharp#206]
Hello, 
i am doing first steps with this API and i am struggling to understand what and where are the data for authentication.

When i create a API key in Coinbase Pro, i give a "Passphrase" and i receive a "API Secret". Could the hex string that stays visible in the API Panel in Coinbase be the "API Key"?

For the authentication i need 3 data fields, "APIKey", "APISecret" and "PassPhrase". 

And also i see in the Docs (from Coinbase) that i need to Sign Requests. Does your API sign requests automatically in each function call?

Thank's in advance.

**Comment by dougdellolio** [comment]
hi @geniosoft 

When you press "+ New API Key" the box titled Passphrase corresponds to the `Passphrase` in the authenticator. After the API key is created by setting permissions and pressing the "Create API Key" button, a box for your API secret will come up (this is only shown once) - this corresponds to the `APISecret` in the authenticator. After you press "Done" you will see a list of the current keys. Use the long string of characters/numbers above the permissions info as the `APIKey`. 

This library handles all of the signing for you for each call so just paste them in as shown.

Let me know if you're still having issues 🤞

**Comment by geniosoft** [comment]
I already did it and with sucess.
In my tries i already did that without sucess, but not knowing what was wrong i tried around and nothing. With your help i understood what was right, and got there.

I already had sucess creating a CoinbaseProClient object and connecting without authentication to the websocket. But each time i used authentication, the tick event did not activated. Knowing what data to put where with your help, i finally understood that the APIKEY in Sandbox was not shared with the main Portal. That was my main error, created a APIKEY in Coinbase and always tested with Sandbox, without authentication it worked, with authentication not worked. 
Only after that i entered in the browser to the Sandbox and realized the APIKEY was not there, created one, commented on the functions and inserted the new values and sucess.

Thank you. I made the first step with sucess. The most difficult, now only 999 more (or a bit more) 👍
